packages/start-plugin-core/src/start-compiler-plugin/compiler.ts

```
@@ -1,12 +1,13 @@
 /* eslint-disable import/no-commonjs */
 import crypto from 'node:crypto'
 import * as t from '@babel/types'
-import { generateFromAst, parseAst } from '@tanstack/router-utils'
-import babel from '@babel/core'
 import {
+  generateFromAst,
+  parseAst,
   deadCodeElimination,
```

ESLint

[error] Member 'deadCodeElimination' of the import declaration should be sorted alphabetically.

(sort-imports)